### PR TITLE
feat: add proxied wallets

### PR DESCRIPTION
**Pull request by Asmadek**
* receive proxied accounts from proxy service and create wallet with proxied account 
* some refactoring for proxy service and model

### CODE REVIEW

File: src/renderer/features/proxies/model/proxies-model.ts [hunk 1]
```diff
@@ -74,11 +115,11 @@ sample({
 
 sample({
   clock: connected,
-  source: walletModel.$accounts,
-  fn: (accounts, chainId) => ({
+  source: { accounts: walletModel.$accounts, proxies: proxyModel.$proxies },
```
Comment (tuul-wq) [inline src/renderer/features/proxies/model/proxies-model.ts:118]:
```suggestion
  source: {
    accounts: walletModel.$accounts,
    proxies: proxyModel.$proxies,
  },
```

File: src/renderer/entities/proxy/model/proxy-model.ts [hunk 2]
```diff
@@ -22,6 +22,11 @@ const removeProxiesFx = createEffect((proxies: ProxyAccount[]) => {
   return storageService.proxies.deleteAll(proxies.map((p) => p.id));
 });
 
+sample({
+  clock: kernelModel.events.appStarted,
```
[the remark this answers is not in the document]
Comment (Asmadek) [reply src/renderer/entities/proxy/model/proxy-model.ts:26]: What I should use in that case? Now `kernelModel.events.appStarted` looks like single option to trigger event to load something from db

File: src/renderer/features/proxies/model/proxies-model.ts [hunk 3]
```diff
@@ -44,21 +60,46 @@ type GetProxiesResult = {
...
+
+  return endpoint.call.getProxies(chainId, nonProxiedAccounts, proxides, proxies) as Promise<GetProxiesResult>;
 });
 
 const disconnectFx = createEffect((chainId: ChainId): Promise<unknown> => {
   return endpoint.call.disconnect(chainId);
 });
 
+const createProxiedesFx = createEffect((proxiedes: PartialProxiedAccount[]): void => {
+  proxiedes.forEach((proxied) => {
```
[the remark this answers is not in the document]
Comment (Asmadek) [reply src/renderer/features/proxies/model/proxies-model.ts]: Please, check `proxiedesCreatedFx` in `walletModel`, is it possible solution or better reimplement `walletCreatedFx` for multiple wallets?

File: src/renderer/entities/proxy/model/proxy-model.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (tuul-wq) [reply src/renderer/entities/proxy/model/proxy-model.ts:26]: Create new event inside `proxy-model` - `proxyStarted`
Inside `index.tsx` call it after `networkModel.events.networkStarted();`
Use it in `sample` 😉

File: src/renderer/features/proxies/model/proxies-model.ts [hunk 4]
```diff
@@ -44,21 +59,48 @@ type GetProxiesResult = {
...
+      } as ProxiedAccount,
+    ];
+
+    return {
+      wallet,
+      accounts,
+    };
+  });
+
+  walletModel.events.proxiedesCreated(newWallets);
```
Comment (tuul-wq) [inline src/renderer/features/proxies/model/proxies-model.ts:101]:
```javascript
sample({
  clock: createProxiedesFx.doneData,
  target: walletModel.events.proxiedesCreated,
});
```

File: src/renderer/features/proxies/model/proxies-model.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (tuul-wq) [reply src/renderer/features/proxies/model/proxies-model.ts]: I really want to separate all wallets creation into their own features
Thus we reduce amount of code inside wallet-model and incapsulate valuable business logic in features
For now I guess it's ok to leave it as is, but changes must be done in the nearest future

**Review by tuul-wq — APPROVED** [review]

**Comment by Asmadek** [comment]
Task linked: [CU-86934j0xd Proxy service: Handle proxy creation: Create proxied wallet](https://app.clickup.com/t/86934j0xd)

**Review by egor0798 — APPROVED** [review]

**Pull request merged**